src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -25,47 +27,136 @@
 
     <Grid Grid.Row="0" Margin="0,10,0,0">
       <Grid.ColumnDefinitions>
-        <ColumnDefinition Width="auto" />
-        <ColumnDefinition Width="auto" />
-        <ColumnDefinition Width="auto" />
         <ColumnDefinition Width="auto" />
         <ColumnDefinition Width="*" MinWidth="20"/>
       </Grid.ColumnDefinitions>
 
-      <nuget:FilterLabel
-        Grid.Column="0"
-        x:Name="_labelBrowse"
-        AutomationProperties.AutomationId="Tab_Browse"
-        Filter="All"
-        Text="{x:Static nuget:Resources.Label_Browse}"
-        ControlSelected="FilterLabel_ControlSelected" />
-      <nuget:FilterLabel
-        Grid.Column="1"
-        x:Name="_labelInstalled"
-        AutomationProperties.AutomationId="Tab_Installed"
-        Filter="Installed"
-        Margin="35,0,0,0"
-        Text="{x:Static nuget:Resources.Label_Installed}"
-        ControlSelected="FilterLabel_ControlSelected">
-      </nuget:FilterLabel>
-      <nuget:FilterLabel
-        Grid.Column="2"
-        x:Name="_labelUpgradeAvailable"
-        AutomationProperties.AutomationId="Tab_Updates"
-        Filter="UpdatesAvailable"
-        Margin="35,0,0,0"
-        Text="{x:Static nuget:Resources.Label_Updates}"
-        ControlSelected="FilterLabel_ControlSelected">
-      </nuget:FilterLabel>
-      <nuget:FilterLabel
-        Grid.Column="3"
-        x:Name="_labelConsolidate"
-        AutomationProperties.AutomationId="Tab_Consolidate"
-        Filter="Consolidate"
-        Margin="35,0,0,0"
-        Text="{x:Static nuget:Resources.Action_Consolidate}"
-        ControlSelected="FilterLabel_ControlSelected"
-        Visibility="Collapsed" />
+      <TabControl x:Name="tabsPackageManagement" IsSynchronizedWithCurrentItem="False" SelectionChanged="TabsPackageManagement_SelectionChanged">
+        <TabControl.Resources>
+          <Style TargetType="TabItem">
+            <Setter Property="Background" Value="{DynamicResource {x:Static nuget:Brushes.HeaderBackground}}" />
+            <Setter Property="FontSize" Value="{Binding FontSize, RelativeSource={RelativeSource AncestorType=UserControl},Converter={StaticResource Font122PercentSizeConverter}}"/>
+            <Setter Property="Foreground" Value="{DynamicResource {x:Static nuget:Brushes.UIText}}" />
```

Comment:
FontSize and Foreground setters may not be doing anything?? (don't see any uses inside the controltemplate below via TemplateBinding) -- could be wrong...should test with and without

Reply:
since the foreground triggers below likely work, i am likely wrong about ForegRound. maybe about fontsize too. could be useful since they are inheritable properties.

Reply:
Before I added Foreground, the color was always white, I think.

FontSize is needed here to utilize the Converter for scaling up the size.

Reply:
i was suggesting it may be worthwhile to confirm they are needed.

Reply:
They are. I rechecked, and the font size/color looks incorrect:

![image](https://user-images.githubusercontent.com/49205731/74787207-7405d280-5263-11ea-8c29-12e403f45464.png)